spec.emu

```diff
@@ -55,9 +35,20 @@ contributors: Jordan Harband
               1. Append code unit U+005C (REVERSE SOLIDUS) to _escapedList_.
               1. Append code unit U+0078 (LATIN SMALL LETTER X) to _escapedList_.
               1. Append code unit U+0033 (DIGIT THREE) to _escapedList_.
+              1. Append _c_ to _escapedList_.
             1. Else if _toEscape_ contains _c_ or _c_ is matched by |WhiteSpace|, then
-              1. Append code unit U+005C (REVERSE SOLIDUS) to _escapedList_.
-            1. Append _c_ to _escapedList_.
+              1. Let _hex_ be Number::toString(𝔽(_c_), 16).
+              1. If the length of _hex_ is 1 or 2, then
+                1. Set _hex_ to StringPad(_hex_, 2, *"0"*, ~start~).
+                1. Append code unit U+0078 (LATIN SMALL LETTER X) to _escapedList_.
+                1. Append the code units in _hex_ to _escapedList_.
+              1. Else,
+                1. Assert: The length of _hex_ is at most 4.
```

Comment:
I'd rather not rely on this. When Unicode adds an SMP whitespace character, I don't want to have to remember to come back here and add support for it. Why don't we pull this out as a generic AO for creating a hex/Unicode escape sequence that uses `\xXX`, `\uXXXX`, and `\u{XXXXXX}` as appropriate?

Reply:
Sounds good to me. Name suggestion?